packages/block-library/src/media-text/media-container.js

```diff
@@ -21,24 +21,26 @@ import { forwardRef } from '@wordpress/element';
 import { isBlobURL } from '@wordpress/blob';
 import { store as noticesStore } from '@wordpress/notices';
 import { media as icon } from '@wordpress/icons';
+export const DEFAULT_FOCAL_POINT = { x: 0.5, y: 0.5 };
 
 /**
  * Constants
  */
 const ALLOWED_MEDIA_TYPES = [ 'image', 'video' ];
 const noop = () => {};
 
-export function imageFillStyles( url, focalPoint ) {
-	return url
-		? {
-				backgroundImage: `url(${ url })`,
-				backgroundPosition: focalPoint
-					? `${ Math.round( focalPoint.x * 100 ) }% ${ Math.round(
-							focalPoint.y * 100
-					  ) }%`
-					: `50% 50%`,
-		  }
-		: {};
+export function imageFillStyles( focalPoint ) {
+	return {
+		objectPosition: focalPoint
+			? `${ Math.round( focalPoint.x * 100 ) }% ${ Math.round(
+					focalPoint.y * 100
+			  ) }%`
+			: `50% 50%`,
+	};
+}
+
+export function focalPosition( { x, y } = DEFAULT_FOCAL_POINT ) {
+	return `${ Math.round( x * 100 ) }% ${ Math.round( y * 100 ) }%`;
 }
```

Comment:
Having both `imageFillStyles` and this new function seems like clutter. Previously `imageFillStyles` was used in a two places and still could be. Or alternatively, `focalPosition` could replace it in both those places. I’d prefer using one or the other and not both.